modules/travel_pay/app/services/travel_pay/client.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module TravelPay
+  class Client
+    ##
+    # HTTP POST call to the VEIS Auth endpoint to get the access token
+    #
+    # @return [Faraday::Response]
+    #
+    def request_veis_token
+      auth_url = Settings.travel_pay.veis.auth_url
+      tenant_id = Settings.travel_pay.veis.tenant_id
+
+      connection(server_url: auth_url).post("/#{tenant_id}/oauth2/token") do |req|
+        req.headers['Content-Type'] = 'application/x-www-form-urlencoded'
+        req.body = URI.encode_www_form(veis_params)
+      end
+    end
+
+    ##
+    # HTTP POST call to the BTSSS token endpoint to get the access token
+    #
+    # @return [Faraday::Response]
+    #
+    def request_btsss_token(veis_token, vagov_token)
+      btsss_url = Settings.travel_pay.base_url
+      api_key = Settings.travel_pay.subscription_key
```

Comment:
These don't seem to be in `settings.yml` - can you add them?

Reply:
We don't want these in `settings.yml` because they are secrets and should not be pushed to Github. This is a larger conversation about local settings vs remote settings. 

Adding time to your calendar today.